.github/workflows/ci.yml

```diff
@@ -5,10 +5,16 @@ on:
   #- cron: "0 10 * * *" # everyday at 10am
   push:
     branches:
-      - "**"
```

Comment:
the `**` currently allow us to push a test docker image for each branch with tag = branch name. This is useful for testing purposes. I think we want to keep this, at least until we have a better workflow for this usecase.

Reply:
suggestions obviously welcome on this matter

Reply:
Seems like a valid approach, I've reverted the commit here 4a808de0617b2201d7cb9412a24c2babf41c50b5 , still keeping a separation of documentation, if you agree?